### PR TITLE
Added broker service as resource type

### PR DESCRIPTION
**Pull request by TheTechArch**
Added Broker service as a resource type

### CODE REVIEW

File: src/Altinn.ResourceRegistry.Core/Enums/ResourceType.cs [hunk 1]
```diff
@@ -27,5 +27,8 @@ public enum ResourceType
 
         [PgName("genericaccessresource")]
         GenericAccessResource = 16,
+
+        [PgName("brokerservice")]
+        BrokerService = 32,
```
Comment (Alxandr) [inline src/Altinn.ResourceRegistry.Core/Enums/ResourceType.cs:32]: Why are these powers of two? Should it be a flags enum?

Comment (Alxandr) [reply src/Altinn.ResourceRegistry.Core/Enums/ResourceType.cs:32]: Also - this isn't feature proof with "search" as is. Cause there is no way to search for services with resourcetype = default.

File: src/Altinn.ResourceRegistry.Core/Enums/ResourceType.cs [hunk 2]
```diff
@@ -14,18 +14,21 @@ public enum ResourceType
         Default = 0,
 
         [PgName("systemresource")]
-        Systemresource = 1,
+        Systemresource = 1 << 1,
```
Comment (Alxandr) [inline src/Altinn.ResourceRegistry.Core/Enums/ResourceType.cs:17]: This changes the value of `Systemresource` to be 2 (and all consecutive values as well). The first one that's non-default should be `1 << 0` (or just `0`, but I tend to do `1 << 0` to keep the pattern).

File: src/Altinn.ResourceRegistry.Core/Enums/ResourceType.cs (hunk 1 above)
Comment (TheTechArch) [reply src/Altinn.ResourceRegistry.Core/Enums/ResourceType.cs]: @Alxandr updated numbering

File: src/Altinn.ResourceRegistry.Core/Enums/ResourceType.cs (hunk 2 above)
Comment (TheTechArch) [reply src/Altinn.ResourceRegistry.Core/Enums/ResourceType.cs]: Dummy me. Fixed it.

**Review by Alxandr — APPROVED** [review]

**Pull request merged**